LemonWay/Models/KycDoc.php

```diff
@@ -76,4 +76,4 @@ function __construct($node) {
 	}
 }
 
-?>
\ No newline at end of file
+?>
```

Comment:
It looks like your IDE added a line feed at the end of the file (which is a good thing!) but this file has a PHP closing tag. It can produce errors if there is characters after the closing tag. http://stackoverflow.com/a/3219401
I think the best solution is to remove the tag `?>` ;)